.env.sample

```diff
@@ -1,3 +1,4 @@
 PROVIDER_URL=http://localhost:8545
 IPFS_GATEWAY_URL=https://ipfs.io
+COVER_ROUTER_URL=https://localhost:5001
```

Comment:
my only concern with hardcoding URLs is we'll have to support the URL indefinitely otherwise we'll break previous SDK versions.

with env vars we can have a migration period to safely migrate users if we do need to make changes to CoverRouter endpoints in the future (i.e. api/v2)

happy to hear what are people thoughts on this